config/puma.rb

```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
```

Comment:
Instead of commenting this out, can you try re-enabling it and adding this line to your `.env.development.local` file
```
WEB_CONCURRENCY=0
```